hoomd/md/constrain.py

```diff
@@ -4,192 +4,49 @@
 # Maintainer: joaander / All Developers are free to add commands for new
 # features
 
-R""" Constraints.
+r""" Constraints.
 
-Constraint forces constrain a given set of particle to a given surface, to have
-some relative orientation, or impose some other type of constraint.
+Constraint forces can constrain particles to be a set distance from each other,
+to have some relative orientation, or impose other types of constraint.
 
 As with other force commands in hoomd, multiple constrain commands can be issued
 to specify multiple constraints, which are additively applied.
 
-Warning: Constraints will be invalidated if two separate constraint commands
-apply to the same particle.
+The `Rigid` class is special in that only one is allowed in a system and is set
+to an `hoomd.md.Integator` object separately in the `rigid` attribute.
+
+Warning:
+    Constraints will be invalidated if two separate constraint commands
```

Comment:
```suggestion
    Constraints will be invalid if two separate constraint forces
```
We don't check for this, so nothing is invalidated - but neither of the constraints will be applied correctly. Also, it's not a command.